### DISCUSSION TITLE
Electron / CJS for nord-theme

### DISCUSSION THREAD

**Discussion by vanjaoljaca** [Milkdown/milkdown#1233] [Q&A]
I tried to setup milkdrop in Electron and got hit with:

Module not found: Error: Package path . is not exported from package /Users/vanjaoljaca/work/vanjacloud/vanjacloud.desktop/node_modules/@milkdown/theme-nord (see exports field in /Users/vanjaoljaca/work/vanjacloud/vanjacloud.desktop/node_modules/@milkdown/theme-nord/package.json)

ChatGPT suggested adding this to theme-nord to make it cjs compatible (needed for electron)

package.json:
```
"exports": {
  ".": {
    "types": "./lib/index.d.ts",
    "import": "./lib/index.es.js",
    "require": "./lib/index.es.js"  // Add this line
  },
  "./style.css": "./lib/style.css"
}
```

This fixed start up at least, but loading the config has no impact on appearance so I think there's more work to be done. Before I dive in I wanted to check in on what is already known about cjs/electron support.

Thanks

**Comment by sytolk** [comment]
I have the same issue style.css from [theme-nord](https://milkdown.dev/docs/api/theme-nord) is not loaded in Electron app. 
@vanjaoljaca do you include tailwind.css in Electron ?
This is milkdown library:
https://github.com/tagspaces/tagspaces-common/tree/develop/packages/tagspaces-md
Usage in Electron app:
https://github.com/tagspaces/tagspaces/blob/develop/package.json#L164

**Comment by sytolk** [comment]
Fixed this issue like this: https://github.com/tagspaces/tagspaces/blob/develop/src/renderer/app.global.css#L10

**Comment by vanjaoljaca** [comment]
yes I include tailwind, do you think tailwind causes it or are you saying its a requirement?

I applied your fix to my index.css, it is this and it still fails with same error:
```
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import 'tailwindcss/utilities';
@import "~@milkdown/theme-nord/style.css";
```
